docs/consume-packages/Central-Package-Management.md

```diff
@@ -0,0 +1,150 @@
+Starting with [NuGet 7.0](..\release-notes\NuGet-7.0.md), you can centrally manage your dependencies in your projects with the addition of a `Directory.Packages.props` file.
+
+The feature is available across all NuGet integrated tooling.
+
+* [Visual Studio 2022 and later](https://visualstudio.microsoft.com/downloads/)
+* [.NET SDK 7.0.000 and later](https://dotnet.microsoft.com/download/dotnet/7.0)
+* [nuget.exe 7.0.0 and later](https://www.nuget.org/downloads)
+
+Older tooling will ignore Central Package Management configurations and features. To use this feature to the fullest extent, ensure all your build environments use the latest compatible tooling versions.
+
+Central Package Management will apply to all project types – including .NET Framework – as long as compatible tooling is used.
+
+## Enabling Central Package Management
+
+To get started with central package management, you can create a `Directory.Packages.props` file at the root of your solution.
+
+Inside, you can define each of the respective package versions required of your solution using `<PackageVersion />` elements that define the package ID and version.
+
+```xml
+<Project>
+  <ItemGroup>
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.1" />
+  </ItemGroup>
+</Project>
+```
+
+Within a project of the solution, you can then use the respective `<PackageReference />` syntax you know and love, but without a `Version` attribute to infer the centrally managed version instead.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Newtonsoft.Json" />
+  </ItemGroup>
+</Project>
+```
+
+Now you're using central package management and managing your versions in a central location!
+
+## Central Package Management rules
+
+The `Directory.Packages.props` file has a number of rules with regards to where it's located in a repository's directory and its context. For the sake of simplicity, only one `Directory.Packages.props` file is evaluated for a given project.
+
+What this means is that if you had multiple `Directory.Packages.props` files in your repository, the file that is closest to your project's directory will be evaluated for it. This allows you extra control at various levels of your repository.
+
+Here's an example, consider the following repository structure:
+
+```
+Repository
+ |-- Directory.Packages.props
+ |-- Solution1
+     |-- Directory.Packages.props
+     |-- Project1
+ |-- Solution2
+     |-- Project2
+```
+
+- Project1 will evaluate the `Directory.Packages.props` file in the `Repository\Solution1\` directory.
+- Project2 will evaluate the `Directory.Packages.props` file in the `Repository\` directory.
+
+## Get started
+
+To fully onboard your repository, consider taking these steps:
+
+1. Create a new file at the root of your repository named `Directory.Packages.props` that declares your centrally defined package versions in.
```

Comment:
ignore my comment above :D The recommendation does apply though. 
Maybe provide some guidance about resolving conflicts, and advertise VersionOverride as a solution, but first encourage using the same version.

Reply:
Good ideas, will add a couple more steps!

Reply:
Did you mean to add more steps here? 

We can always improve the doc later. Maybe we can enhance during docs days, as long as we have an issue to know what we need to do that.